src/routes/admin/[classId]/+page.svelte

```
@@ -256,7 +256,7 @@
 </script>
 
 <div class="min-h-screen p-8 max-w-7xl mx-auto">
-	<a class="btn mb-4 btn-ghost" href="/admin"><ArrowLeft size={16} />Back</a>
+	<a class="btn mb-4 btn-ghost rounded-full" href="/admin"><ArrowLeft size={16} />Back</a>
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)